src/main/resources/data.sql

```diff
@@ -1,30 +1,30 @@
-insert into event values ( 8, 'Music', 'halloween', '2020-10-31T20:00:00', file_read('image_event_1.jpg'), 'zaal', 'halloween', 15.00 );
-insert into event values ( 9, 'Music', 'nieuwe schooljaar', '2020-09-01T08:20:00', file_read('image_event_1.jpg'), 'school', 'nieuwe schooljaar', 765.00 );
-insert into event values ( 10, 'Music', 'olympische spelen openingsceremonie', '2020-07-24T20:00:00', file_read('image_event_1.jpg'), 'Tokyo', 'olympische spelen', 80.00 );
-insert into event values ( 11, 'Music', 'olympische spelen slotceremonie', '2020-07-24T20:00:00', file_read('image_event_1.jpg'), 'Tokyo', 'olympische spelen', 80.00 );
-insert into event values ( 12, 'Music', 'de presentatie van deze website', '2020-06-15T00:00:00', file_read('image_event_1.jpg'), 'Tokyo', 'eindproef', 0.00 );
-insert into event values ( 13, 'Music', 'amerikaanse thanksgiving', '2020-11-26T14:00:00', file_read('image_event_1.jpg'), 'bij mama', 'thanksgiving', 15.00 );
-insert into event values ( 14, 'Theatre', 'black friday', '2020-11-27T00:00:00', file_read('image_event_1.jpg'), 'bij mama', 'black friday', 0.00 );
-insert into event values ( 15, 'Theatre', 'wapensstilstand', '2020-11-11T11:11:11', file_read('image_event_1.jpg'), 'kerkhof', 'wapensstilstand', 0.00 );
-insert into event values ( 16, 'Theatre', 'star ways day', '2020-05-04T05:04:00', file_read('image_event_1.jpg'), 'kelder', 'star wars day', 0.00 );
-insert into event values ( 17, 'Theatre', 'presidentiele verkiezingen vs', '2020-11-03T00:00:00', file_read('image_event_1.jpg'), 'kieshokjes', 'presidentiele verkiezingen vs', 30.00 );
-insert into event values ( 18, 'Theatre', 'Oracle''s annual convention brings together IT management, business decision-makers and line-of-business end users. Typically, the conference includes keynotes from leadership at Oracle and from other partner organizations. More 2,500 additional sessions and workshops focus on other IT and business-related topics.', '2020-09-21T10:00:00', file_read('image_event_1.jpg'), 'San Francisco', 'Oracle OpenWorld', 300.00 );
-insert into event values ( 19, 'Theatre', 'AWS Re:invent is Amazon''s opportunity to update IT and business leaders on the latest features of its cloud service.The event features keynote announcements, training and certification opportunities, access to more than 2,000 technical sessions, a partner expo, and more.', '2020-11-30T11:00:00', file_read('image_event_1.jpg'), 'Las Vegas', 'AWS re:Invent', 30.00 );
-insert into event values ( 20, 'Theatre', 'not the music one', '2020-08-06T00:00:00', file_read('image_event_1.jpg'), 'Las Vegas', 'Def Con 28', 320.00 );
+/* values (id, category, description, datetime, image, location, name, price, youtubeUrl) */
+insert into event values ( 1, 'Music', 'test description', '2020-12-20T20:30:54', file_read('image_event_1.jpg'), 'genk', 'concert', 20.00, 'https://www.youtube.com/embed/3UsKYsLSGpU');
```

Comment:
tests probably don't run because you should define: insert into event (id, category,...) values (1, 'Music',...)